rollup.utils.js

```diff
@@ -298,4 +350,9 @@ module.exports = {
  *   sourceDir: string;
  *   version: string;
  * }} BuildInfo
+ *
+ * @typedef {{
+ *   vars?: Record<string, string[]>;
+ *   types?: Record<string, string[]>;
+ * }} MagicExports
```

Comment:
More in line with the actual `importKind` would be to use `value` & `type`, like we do in `packageExports` of the `replace-remix-imports` migration & of the ESLint warning

https://github.com/remix-run/remix/blob/ec49faa9d8155542e157dbac7977393003f0c7a7/packages/remix-dev/cli/migrate/migrations/replace-remix-imports/transform/mapNormalizedImports/packageExports.ts#L27-L30
https://github.com/remix-run/remix/blob/ec49faa9d8155542e157dbac7977393003f0c7a7/packages/remix-eslint-config/rules/packageExports.js#L1-L4

```suggestion
 * @typedef {{
 *   value?: Record<string, string[]>;
 *   type?: Record<string, string[]>;
 * }} MagicExports
```